task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -49,17 +49,20 @@ spec:
         chmod +x opm grpc_health_probe
 
         if ! ./opm version; then
```

Comment:
@jsztuka Should we count the the number of the `if` checks as the number of `successes` and `failures` in `HACBS_TEST_OUTPUT`?

Reply:
I'm not Jiri but +1 from me :)

Reply:
Thanks :)

Reply:
I would say we should.